tests/integration/staking/keeper/grpc_query_test.go

```diff
@@ -29,7 +29,7 @@ func createValidatorAccs(t *testing.T, f *fixture) ([]sdk.AccAddress, []types.Va
 	sortedVals := make([]types.Validator, len(validators))
 	copy(sortedVals, validators)
 	hi := types.NewHistoricalInfo(header, sortedVals, f.stakingKeeper.PowerReduction(f.sdkCtx))
-	assert.NilError(t, f.stakingKeeper.SetHistoricalInfo(f.sdkCtx, 5, &hi))
+	assert.NilError(t, f.stakingKeeper.HistoricalInfo.Set(f.sdkCtx, types.GetHistoricalInfoKeyWithoutPrefix(5), hi))
```

Comment:
basically these instances of `types.GetHistoricalInfoKeyWithoutPrefix(key)` become: `uint64(key)`